packages/core/src/components/HorizontalScrollGrid/HorizontalScrollGrid.test.tsx

```diff
@@ -89,9 +91,21 @@ describe('<HorizontalScrollGrid />', () => {
     fireEvent.click(rendered.getByTitle('Scroll Left'));
     expect(lastScroll).toBeLessThan(0);
 
-    delete HTMLElement.prototype.scrollLeft;
-    delete HTMLElement.prototype.offsetWidth;
-    delete HTMLElement.prototype.scrollWidth;
-    delete HTMLElement.prototype.scrollBy;
+    // reset to original values
+    Object.defineProperties(HTMLElement.prototype, {
```

Comment:
Since the `originalPrototypes` is a reference to the same object, this won't actually do much.

I'd check if it's possible to switch this to use `jest.spyOn` with a `'get'` accessor instead. Docs on that [here](https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname-accesstype)

Otherwise we can ofc also just grab references to the original property values instead.